packages/sacado/CMakeLists.txt

```diff
@@ -91,6 +91,11 @@ TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_VIEW_SPEC
   "Enable Sacado specializations for Kokkos::View<...>"
   ON )
 
+TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_GTest
```

Comment:
Where is `${PACKAGE_NAME}_ENABLE_GTest` defined?  GTest is not listed in `LIB_OPTIONAL_DEP_TPLS` at:

* https://github.com/trilinos/Trilinos/blob/24715b8221e5221e9c4e3fe97c78953b258df07b/packages/sacado/cmake/Dependencies.cmake#L6

and I think it should.  Otherwise, I don't see how `Sacado_ENABLE_GTest` will ever get set to `ON`.  See:

* https://tribits.org/doc/TribitsDevelopersGuide.html#how-to-add-a-new-tribits-tpl-dependency

Reply:
It's not a TPL dependency.  It turns on/off whether Sacado's internal GTest library is built, along with the corresponding tests that use it.

Reply:
Wait, Sacado has its own GTest implementation?  Why not just use the GTest package already there?

Reply:
Yes.  It was added in PR #7736.  Trilino's GTest package is too old and doesn't include all of the needed functionality.  I attempted to upgrade it in that PR, but that was unsuccessful (see issue #7698 for more details).  Basically it amounted to a complete overhaul of the Trilinos GTest package, and I could never get the installed CMake files to work correctly (even though the autotester passed with it, codes like Albany wouldn't configure correctly).